src/app/custom-keyboard/_components/BoardOption.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import classNames from 'classnames/bind';
+import { useContext } from 'react';
+import { KeyboardDataContext } from '@/context/customKeyboardContext';
+import { HexColorPicker } from 'react-colorful';
+import { CustomKeyboardTypeTypes, CustomKeyboardTextureTypes } from '@/types/CustomKeyboardTypes';
+import styles from './BoardOption.module.scss';
+
+const TYPE_BUTTONS = [
+  { type: 'full', name: '풀 배열', price: 35000 },
+  { type: 'tkl', name: '텐키리스', price: 30000 },
+];
+
+const TEXTURE_BUTTONS = [
+  { texture: 'metal', name: '금속', price: 35000 },
+  { texture: 'plastic', name: '플라스틱', price: 30000 },
+];
+
+const cn = classNames.bind(styles);
```

Comment:
```suggestion
const cn = classNames.bind(styles);
```
선언하시는 위치가 컴포넌트마다 달라지는데 항상 같은 위치에 놓는게 더 좋은 방향이라 생각합니다